New_PlatformGame/Assets/ScaffoldingScript.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking.PlayerConnection;
+
+public class ScaffoldingScript : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public GameObject coinPrefab;
+    public const int percentage = 30;
```

Comment:
```suggestion
    public const int percentage = 30;
```
const를 통한 상수화의 좋은 예시. But, 상수는 일반적인 변수와는 다른 고정된 값이기 때문에 set를 할 수 없으므로 타 변수와는 다르다는 것을 알려주기 위해 코드 컨벤션에서 상수는 ~~하게 작성해요 라고 지정해두는 편이 좋다.